ext/pgsql/pgsql.c

```diff
@@ -4448,7 +4245,6 @@ PHP_FUNCTION(pg_escape_bytea)
 }
 /* }}} */
 
-#ifndef HAVE_PQUNESCAPEBYTEA
```

Comment:
Wrong way around? This code should be dropped.

Reply:
Oops. Thanks!